meter/tesla.go

```diff
@@ -39,7 +46,9 @@ type teslaBatteryResponse struct {
 // Tesla is the tesla powerwall meter
 type Tesla struct {
 	*request.Helper
-	uri, usage string
+	uri      *url.URL
```

Comment:
any reason to not use the string here? It's the standard pattern in evcc.

Reply:
I had the problem that the url must not contain a trailing slash, otherwise everything would fail. I think it would be best to handle both cases (with and without trailing slashes). The easiest way to do so would be using a URL struct.